assets/js/theme/common/faceted-search.js

```diff
@@ -56,6 +59,15 @@ class FacetedSearch {
         this.collapsedFacets = [];
         this.collapsedFacetItems = [];
 
+        this.onMoreOptionsOpened = (event) => {
+            const $filterItems = $(event.target).find('#facetedSearch-filterItems');
+            if ($filterItems.length) {
+                this.options.modal.setupFocusableElements(SHOW_MORE_OPTIONS);
+            }
+        };
+
+        $('body').on(opened, this.onMoreOptionsOpened);
```

Comment:
Do we need to bind onMoreOptionsOpened on every modal which is open in body at pages with faceted search?

Reply:
Do you mean to replace onMoreOptionsOpened just on modal where options can appear?

Reply:
Could we do it?

Reply:
yes -its done